conda/cli/conda_argparse.py

```diff
@@ -1058,7 +1058,10 @@ def configure_parser_package(sub_parsers):
 
 
 def configure_parser_remove(sub_parsers, aliases):
-    help_ = "Remove a list of packages from a specified conda environment."
+    help_ = (
+        "Remove a list of packages from a specified conda environment."
+        + " Use `--all` flag to remove all packages and the environment itself."
```

Comment:
[Adjacent strings will auto concatenate](https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation):

```suggestion
        "Remove a list of packages from a specified conda environment. "
        "Use `--all` flag to remove all packages and the environment itself."
```